heon/compare_flow_sequence.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+
+
+def compare_lists(list1, list2):
+    lists_are_equal = True
+    i = 0
+    while i < len(list1):
```

Comment:
If i recall correctly, you can use `list1 == list2` directly to compare two lists in Python. 

BTW this code may also break if list1 is longer than list2. The loop counter `i` may reach an index where the access `list2[i]` is out of range.

Reply:
Thanks, I will modify the function as well as make it call sys.exit() on different list lengths.